src/framework/components/element/text-element.js

```diff
@@ -1160,6 +1185,169 @@ Object.assign(pc, function () {
         }
     });
 
+    Object.defineProperty(TextElement.prototype, "outlineColor", {
```

Comment:
We need to define these in the element and ensure they are initialized in initializeComponent in ElementComponentSystem.